features/react_native.feature

```diff
@@ -11,3 +11,80 @@ Scenario: React Native sends requests
     And 1 requests are valid for the android mapping API and match the following:
       | versionCode | versionName | appId                     |
       | 1           | 1.0         | com.bugsnag.android.rnapp |
+
+    And 1 requests are valid for the JS source map API and match the following:
+        | appVersionCode | appVersion | overwrite | dev   |
+        | 1              | 1.0        | false     | false |
```

Comment:
All of these tests use the same `appVersionCode` & `appVersion` — is it worth having a test with different values, to prove that it picks them up correctly?

Reply:
I can change the fixture values to more 'unusual' values, but the parsing of appVersionCode/versionCode from the manifest is already covered by E2E tests elsewhere